### DISCUSSION TITLE
Themes for custom webcomponent

### DISCUSSION THREAD

**Discussion by chiragSAP** [UI5/webcomponents#7296] [Q&A] [answered]
Hi,

I have created a custom webcomponent package using ui5-webcomponents documentation. For the theme specific css I have created parameters-bundles.css file inside src/themes/sap_horizon_dark and changed the value of variable declared in the central css file.

When the webcomponent is consumed in React app, the changes made for the sap_horizon_dark theme are not visible when the theme is applied. Is there anything else to be done for the component to adapt according to the theme?

Best regards,
Chirag

**Comment by ilhan007** [comment, marked as answer]
Hello @chiragSAP 
is there an error in the console? 

Usually to enable theming (e.g. using themes other than the default) for the standard components you need to import the theming assets`@ui5/webcomponents/dist/Assets.js` as described here. 
Same goes for the custom components packages. If you used the @ui5/webcomponents-tools, you also have Assets.js file for your package. The React app that consumes your component package should import it.
I can only think of this based on the provided information.

BR,
ilhan
